packages/uui-color-picker/lib/uui-color-picker.element.ts

```diff
@@ -176,11 +186,7 @@ export class UUIColorPickerElement extends LabelMixin('label', LitElement) {
   connectedCallback(): void {
     super.connectedCallback();
 
-    if (this.value) {
-      this.setColor(this.value);
-    } else {
-      this.setColor('#000');
-    }
+    this.setColor(this.value);
```

Comment:
You already call `this.setColor` in `set value()`, so consider if it is necessary to call this again.